docker-compose.yaml

```diff
@@ -1,14 +1,38 @@
 services:
```

Comment:
Would be nice to add a [`name:` tag](https://docs.docker.com/compose/how-tos/project-name/) to each of the composes so that Docker automatically names the stack and so forth in an organized manner.

Even if we just set it to the name of the directory it is in, having it be explicit is IMO a good thing. At work I see too many stacks on people's machines named "api" because that's the folder that usually contains the compose :rofl: